Dockerfile

```diff
@@ -22,7 +22,7 @@ RUN tar -xzf $kafka_distro
 RUN rm -r kafka_$scala_version-$kafka_version/bin/windows
 
 
-FROM eclipse-temurin:17-jre-alpine
+FROM eclipse-temurin:17.0.3_7-jre
```

Comment:
Why are we switching back from alpine?

Reply:
If you look at https://hub.docker.com/_/eclipse-temurin apparently they publish updates for alpine platform less frequently. Also, in the https://github.com/adoptium/adoptium/issues/140 is behind other platforms in the list. While I was monitoring this list saw the platforms were handled in this order.